tools/hny_dataset_cleanup_tool/hny-column-cleanup.py

```diff
@@ -198,10 +202,16 @@ def delete_columns(dataset, api_key, api_url, is_dry_run, column_ids):
             parser.error('--date YYYY-MM-DD is required when using --mode ' + args.mode)
 
         if len(columns_to_delete.keys()) > 0:
+            if not args.dry_run:
+                print("WARNING: Dry run disabled - this will delete live columns!")
```

Comment:
hey @andy-focusedlabs !   I like the other output changes but this one seems confusing to me.  Dry-run mode isn't the default, and so I think WARNING people that the tool is doing the expected thing isn't quite right here.